bokeh/_json_encoder.py

```diff
@@ -79,4 +79,4 @@ def default(self, obj):
             return self.transform_python_types(obj)
 
 def serialize_json(obj, encoder=BokehJSONEncoder, **kwargs):
-    return json.dumps(obj, cls=encoder, allow_nan=False, **kwargs)
+    return json.dumps(obj, cls=encoder, allow_nan=True, **kwargs)
```

Comment:
> The json reviver is a valid solution for now.

So do you have any suggestions of a better solution, perhaps not feasible right now?